### PR TITLE
Add Number to default extensions

### PR DESCRIPTION
**Pull request by kaicataldo**
Fixes https://github.com/adobe-community/issues/issues/9881

### CODE REVIEW

File: test/behavior/dynamic.js [hunk 1]
```diff
@@ -199,7 +199,7 @@ describe('dynamic rules', function() {
 
         fox.process({
           foo: [
-            { id: 3, width: 100 },
+            { id: 3, width: 100.6 },
```
Comment (kaicataldo) [inline test/behavior/dynamic.js]: Hey look, it works!

**Comment by designjockey** [comment]
👍

**Comment by jackie** [comment]
🎉

**Comment by jackie** [comment]
👍

**Pull request merged**